entity.go

```diff
@@ -27,6 +27,9 @@ type Entity interface {
 	// Imports includes all external files required by this entity.
 	Imports() []File
 
+	// Dependents includes all entities that require this entity.
```

Comment:
Clarify that this accessor only returns results if the right generation code from above is used.